mlflow/utils/logging_utils.py

```diff
@@ -1,6 +1,39 @@
-from __future__ import print_function
 import sys
+import logging
+import logging.config
 
+import mlflow
 
-def eprint(*args, **kwargs):
-    print(*args, file=sys.stderr, **kwargs)
+
+LOGGING_LINE_FORMAT = "%(asctime)s %(levelname)s %(filename)s:%(lineno)d: %(message)s"
+LOGGING_DATETIME_FORMAT = "%Y/%m/%d %H:%M:%S"
+
+DEFAULT_LOGGER_NAME = mlflow.__name__
+
+
+def _configure_default_logger():
+    logging.config.dictConfig({
+        'version': 1,
+        'disable_existing_loggers': False,
+        'formatters': {
+            'standard': {
+                'format': LOGGING_LINE_FORMAT,
+                'datefmt': LOGGING_DATETIME_FORMAT,
+            },
+        },
+        'handlers': {
+            'default': {
```

Comment:
The word 'default' here seems suspicious. Should we pick something like "mlflow"?

Reply:
Changed to `mlflow_handler`